### DISCUSSION TITLE
The problem has been solved

### DISCUSSION THREAD

**Discussion by Hello-Linux** [apache/skywalking#9616] [Ideas]
The problem has been solved

**Comment by wu-sheng** [comment]
All discussion must be in English.
SkyWalking doesn't have so call official support. Features are all from contributor.

Please update all in English, otherwsie I have to delete or close this.

**Comment by wu-sheng** [comment]
SkyWalking issue list is used for tracking things we have ppl working on.
If this is an asking, rathee than you are going to do. This should move to discussion panel.

Like I mentioned, we are different from many Chinese initialized projects. We believe meaningful enhancements would be contributed by community. If not, than it is not that useful to drive ppl to start.

**Reply by wu-sheng** [reply]
This is x-request ID as trace ID. Nothing about logs

**Reply by wu-sheng** [reply]
I don't know. I didn't do anything like this.

**Reply by wu-sheng** [reply]
Why you keep ping me? I made myself very clear.
There is no reason, because no one took an action.

**Reply by wu-sheng** [reply]
I am not going to follow this thread anymore. You treat me as customer support, which is not anyone's role in the community.
